.github/workflows/release-code-freeze.yml

```
@@ -80,13 +80,22 @@ jobs:
   slack-notification:
     name: "Send notification to Slack"
     needs: [check-code-freeze, create-release-pr]
-    if: ${{ ! ( inputs.skipSlackPing && needs.create-release-pr.outputs.release-pr-id ) }}
+    if: ${{ ! ( inputs.skipSlackPing && cancelled() ) }}
```

I think condition here needs to be inverted:
```suggestion
    if: ${{ inputs.skipSlackPing || cancelled() }}
```